### PR TITLE
Delayed init getFeature plugin

### PR DESCRIPTION
**Pull request by llienher**
Fix https://jira.camptocamp.com/browse/GEO-844

I did not tested it yet.

### CODE REVIEW

File: core/src/script/CGXP/plugins/GetFeature.js [hunk 1]
```diff
@@ -258,7 +258,14 @@ cgxp.plugins.GetFeature = Ext.extend(gxp.plugins.Tool, {
         if (this.activateToggleGroup) {
             cgxp.plugins.ToolActivateMgr.register(this);
         }
-        this.buildControls(target.mapPanel.map);
+      var that = this;
+      setTimeout((function() {
```
Comment (sbrunner) [inline core/src/script/CGXP/plugins/GetFeature.js]: Take care on the alignment ... it makes difficult to read

File: core/src/script/CGXP/plugins/GetFeature.js [hunk 2]
```diff
@@ -258,7 +258,14 @@ cgxp.plugins.GetFeature = Ext.extend(gxp.plugins.Tool, {
...
+            // Initializing control AFTER baselayers (and projection) have
+            // been initialized in the map and map projection is set,
+            // to solve some weird behavior with Mapserver when the snsName
+            // attribute is not set in the wfs request.
+            that.buildControls(target.mapPanel.map);
+        }), 500);
```
Comment (sbrunner) [inline core/src/script/CGXP/plugins/GetFeature.js]: Did you try with 0?

**Comment by ochriste** [comment]
need to discuss this with @sbrunner

**Comment by ochriste** [comment]
as a quick test (not that this should be the solution, but worth a try):
try to also add a delay in the wfspermalink initialization

**Comment by jwkaltz** [comment]
We can close this PR: there is no easy solution and customer agrees to leave the problem as a known issue until migration to ngeo is done.

**Pull request closed without merging**